integration-tests/test-cases/scenario-tests/src/test/resources/basic.xml

```diff
@@ -30,6 +30,19 @@
                     <include name="invoke"/>
                     <include name="terminate"/>
                     <include name="repeatTerminate"/>
+                    <include name="deleteImage"/>
+                </methods>
+            </class>
+            <class name="io.cellery.integration.scenario.tests.EmployeePortalTestCase">
```

Comment:
Can't we use `<test name="...." parallel="false">` instead of defining all the methods?

Reply:
Updated.